src/shogun/classifier/svm/OnlineSVMSGD.cpp

```diff
@@ -209,14 +209,14 @@ void COnlineSVMSGD::init()
 	SG_ADD(&C1, "C1", "Cost constant 1.", ParameterProperties::HYPER);
 	SG_ADD(&C2, "C2", "Cost constant 2.", ParameterProperties::HYPER);
 	SG_ADD(&lambda, "lambda", "Regularization parameter.", ParameterProperties::HYPER);
-	SG_ADD(&wscale, "wscale", "W scale");
-	SG_ADD(&bscale, "bscale", "b scale");
-	SG_ADD(&epochs, "epochs", "epochs");
+	SG_ADD(&wscale, "wscale", "W scale", ParameterProperties::HYPER);
+	SG_ADD(&bscale, "bscale", "b scale", ParameterProperties::HYPER);
+	SG_ADD(&epochs, "epochs", "epochs", ParameterProperties::HYPER);
```

Comment:
I couldn't tell what this epochs parameter does? is that the number of epochs to run? or is it the number of epochs as online training evolves?

Reply:
Think latter so hyper is good